### PR TITLE
build: add repo archive target

### PR DESCRIPTION
**Pull request by cbuto**
This PR adds a target to generate a repo archive for use in air gapped deployments.

### CODE REVIEW

**Review by branden — APPROVED** [review]

File: make/release.mk [hunk 1]
```diff
@@ -1,26 +1,39 @@
-IMAGE_BUILD_DIR := _build
-IMAGE_TAR_FILE := $(IMAGE_BUILD_DIR)/catalog-applications-image-bundle.tar.gz
-CATALOG_IMAGES_TXT := $(IMAGE_BUILD_DIR)/catalog_images.txt
+BUILD_DIR := _build
+IMAGE_TAR_FILE := $(BUILD_DIR)/catalog-applications-image-bundle.tar.gz
```
Comment (gracedo) [inline make/release.mk:2]:
```suggestion
IMAGE_TAR_FILE := $(BUILD_DIR)/dkp-catalog-applications-image-bundle.tar.gz
```
do we want to prefix with dkp- to make it clear where these images are from?

File: make/release.mk [hunk 2]
```diff
@@ -1,26 +1,39 @@
...
+REPO_ARCHIVE_FILE := $(BUILD_DIR)/dkp-catalog-applications.tar.gz
+CATALOG_IMAGES_TXT := $(BUILD_DIR)/catalog_images.txt
```
Comment (gracedo) [inline make/release.mk:4]: same here maybe?

File: make/release.mk (hunk 1 above)
Comment (cbuto) [reply make/release.mk]: yep done in 61b36c138e6a0920e6e3b265a1b4d966ce957e0b, this file will only be named this locally so the docs are still good

File: make/release.mk (hunk 2 above)
Comment (cbuto) [reply make/release.mk]: yep done in 61b36c138e6a0920e6e3b265a1b4d966ce957e0b as well

**Review by cwyl02 — APPROVED** [review]

**Pull request merged**